src/uct/api/uct.h

```diff
@@ -1909,8 +1913,8 @@ UCT_INLINE_API ucs_status_ptr_t uct_ep_tag_rndv_zcopy(uct_ep_h ep, uct_tag_t tag
  * @ingroup UCT_TAG
  * @brief Cancel outstanding rendezvous operation.
  *
- * The transport will disregard the outstanding operation without calling
- * completion callback provided in @ref uct_ep_tag_rndv_zcopy.
+ * This routine makes transport disregard the outstanding operation without
```

Comment:
makes -> signals the underlaying transport ?
What happens if the callback was already called ?